src/ui/browserAction/popupPage.js

```diff
@@ -51,30 +52,36 @@ export class BrowserActionPopup extends LitElement {
     pageURL: { type: String },
     _siteContext: { type: Object },
     hasSiteContext: { type: Boolean },
+    _siteContexts: { type: Array },
   };
 
   constructor() {
     super();
     this.pageURL = null;
     this._siteContext = null;
-    browser.tabs.onUpdated.addListener(() => this.updatePage());
-    browser.tabs.onActivated.addListener(() => this.updatePage());
-    vpnController.state.subscribe((s) => (this.vpnState = s));
+    vpnController.state.subscribe((s) => {
+      this.vpnState = null;
```

Comment:
Why nulling it?

Reply:
we were nulling it in `updatePage()` but maybe we don't actually need to do this any longer. investigating.

Reply:
Removing...